src/PrestaShopBundle/Command/LegacyLinkLinterCommand.php

```diff
@@ -57,6 +57,34 @@ class LegacyLinkLinterCommand extends Command
         'admin_common_sidebar',
         'admin_common_reset_search',
         'admin_common_reset_search_by_filter_id',
+        'admin_products_combinations_delete_combination',
+        'admin_products_combinations_bulk_delete',
+        'admin_products_attribute_groups',
+        'admin_all_attribute_groups',
+        'admin_products_combinations_generate',
+        'admin_products_specific_prices_list',
+        'admin_products_specific_prices_create',
+        'admin_products_specific_prices_edit',
+        'admin_products_specific_prices_delete',
+        'admin_products_light_list',
+        'admin_products_search',
+        'admin_products_reset_grid_search',
+        'admin_products_grid_category_filter',
+        'admin_products_grid_shop_previews',
+        'admin_products_download_virtual_product_file',
+        'admin_products_delete_from_shop_group',
+        'admin_products_delete_from_shop',
+        'admin_products_enable_for_all_shops',
+        'admin_products_enable_for_shop_group',
+        'admin_products_disable_for_all_shops',
+        'admin_products_disable_for_shop_group',
+        'admin_products_search_associations',
+        'admin_products_search_combinations',
+        'admin_products_search_product_combinations',
+        'admin_products_quantity',
+        'admin_categories_get_categories_tree',
+        'admin_catalog_price_rules_list_for_product',
+        'admin_features_horizontal_index'
```

Comment:
```suggestion
        'admin_features_horizontal_index',
```

CS fixer is complaining about this missing comma :+1: